data/human/human missions.txt

```diff
@@ -107,7 +107,269 @@ mission "Pookie, Part 2"
+			choice
+				`	(Get out before it gets ugly.)`
+					goto nopelate
+				`	(Stay for a while longer.)`
+			`	Continuing to look on, you hear a loud bang to your right as the stagehand gasps in pain and collaspes. You turn to see that a mustached man next to you has his gun raised. Before you can react, you hear another gunshot and see a spurt of blood erupt from the man's temple. Almost instantly, the entire audience is up in arms as gunfire comes in from every direction. Acting instinctually, you duck down and crawl through the forest of legs towards the exit. Once there, you reach your hand up and push the door open before running out.`
+				goto outside
+			label nopelate
+			`	As you turn around, you hear a volley of gunshots from behind, and decide to run out of the building, tackling the door open.`
+				goto outside
+			label nope
+			`	You head towards the exit, pushing aside audience members completely captivated by the fight. Flinging the door open, you hear a gunshot from behind, and decide to run out.`
+			label outside
+			`	As you step outside, you are greeted by sheets of rain lashing you. The strength of the torrent almost causes you to fall backwards, but you manage to steady yourself. You take a glance back at the racing complex to find that the guard in the black uniform is missing. You turn back to begin the trek back to your ship, but halfway there, you feel something touch your lower leg.`
+			choice
+				`	(Turn around.)`
+					goto turner
+				`	(Run away.)`
+			`	You begin to run, but slip in the mud and fall on your face. You roll over to come face-to-face with a dour-looking dog eyeing you. Sitting up, you recognize the poodle as Puny, who must have snuck out of the complex during all the commotion. As you get off of the ground, Puny looks up at you with puppy-dog eyes.`
+				goto kindly
+			label turner
+			`	You turn around and come face-to-face with a dour-looking dog looking up at you with puppy-dog eyes. Inspecting it, you recognize the poodle as Puny, who must have snuck out of the complex during all the commotion.`
+			label kindly
+			choice
+				`	(Pet her.)`
+				`	(Shoo her.)`
+					goto shoo
+			`	You crouch down and rub the soggy tufts of hair on Puny's head. As you continue, a smile creeps onto her face, beaming in spite of the soaking rain. You think about what to do with her.`
+			choice
+				`	(Bring Puny aboard your ship.)`
+				`	(Shoo Puny off.)`
+					goto shoo
+			`	You pick up Puny and carry her onboard. As you dry yourselves, you think back to what the announcer said about Puny being owned by someone from the Paradise Worlds, and wonder where you might be able to drop her off.`
+			branch remember
+				not "bet on hound racing"
+			`	As you step into the cockpit, you recall the bet you put on the hound race. Looks like you won't be getting your money back.`
+			label remember
+			``
```

Comment:
If we don't want the blank line at the end of the conversation, then we could do something like this:
```suggestion
			`	As you step into the cockpit, you recall the bet you put on the hound race. Looks like you won't be getting your money back.`
				decline
			label remember
			`	<Insert something here about being lucky you didn't spend any money on a bet or something. Anything to sound off the mission in some way similar to the above line.>`
```